programs/lfx-mentorship/2023/02-Jun-Aug/README.md

```diff
@@ -26,4 +26,310 @@ Mentee application instructions can be found on the [Program Guidelines](https:/
 
 ---
 
+Table of Contents
+
+* [Armada](#armada)
+* [CoreDNS](#coredns)
+* [Jaeger](#jaeger)
+* [Knative](#knative)
+* [Kubescape](#kubescape)
+* [Kyverno](#kyverno)
+* [Notary](#notary)
+* [Meshery](#meshery)
+* [ORAS](#oras)
+* [Tetragon](#tetragon)
+* [WasmEdge](#wasmedge)
+* [Konveyor](#konveyor)
+* [Strimzi](#strimzi)
+* [Thanos](#thanos)
+* [KubeArmor](#kubearmor)
+* [LitmusChaos](#litmuschaos)
+* [Vitess](#vitess)
+* [KubeVela](#kubevela)
+* [Service Mesh Performance](#service-mesh-performance)
+
+---
+
+## Accepted projects
+
+### Armada
+
+#### Build interfaces around Postgres for Armada
+
+- Description: Open source projects should not be hard coded to a particular Database. Armada currently only allows users to use Postgres. This project is to build interfaces around our connections to Postgres so we can allow other databases.
+- Expected outcomes:
+  - A interface is created that allows Armada to interact with any SQL database without exposing implementation details of postgres
+  - increase Test coverage
+- Recommend Skills: Go, SQL
+- Mentor(s):
+  - Geoffrey Wilson, @suprjinx, geoff@gr-oss.com
```

Comment:
```suggestion
  - Geoffrey Wilson, @suprjinx, geoff@gr-oss.io
```

https://github.com/cncf/mentoring/pull/985

Reply:
I've updated the project as well.